dali/pipeline/data/types.h

```diff
@@ -264,6 +264,21 @@ class DLL_PUBLIC TypeInfo {
   DLL_PUBLIC void Copy(void *dst, const void **srcs, const Index *sizes, int n, cudaStream_t stream,
                        bool use_copy_kernel = false) const;
 
+  /**
+   * @brief Copies from SrcBackend contiguous buffer to DstBackend scattered locations
+   * @param dsts destination pointers
+   * @param src source pointer
+   * @param sizes number of elements for each of the pointers specified in dsts
+   * @param n number of copies to process
+   * @param stream CUDA stream. Only used in copies to/from GPU backend or from/to Host
+   *        pinned memory when use_copy_kernel is true
```

Comment:
Can't the `cudaMemcpyAsync` be performed also on this stream?

Reply:
Probably my phrasing wasn't the best here. It means it's used when copying to/from GPU or Host pinned memory. I'll try to reword it